test/functional/groups_controller_test.rb

```diff
@@ -10,42 +10,43 @@ class GroupsControllerTest < ActionController::TestCase
   end
 
   test "should get index" do
-    get :index
+    get groups_path
     assert_response :success
     assert_not_nil assigns(:groups)
   end
 
   test "should get new" do
-    get :new
+    get new_group_path
     assert_response :success
   end
 
   test "should create group" do
     assert_difference('Group.count', 1) do
-      post :create, params: {group: @group.attributes, supergroup_id: groups(:district_zg).id}
+      # post groups_url, params: {group: @group.attributes, supergroup_id: groups(:district_zg).id}
```

Comment:
To rozumiem tak specjalnie? Bo widzę, że działa w obu przypadkach.

Reply:
Nie pamiętam! Poprawiłem na coś bardziej sensownego.

Reply:
Nie pamiętam! Poprawiłem na coś bardziej sensownego.